src/test/java/edu/indiana/dlib/amppd/util/TestHelper.java

```diff
@@ -237,6 +237,7 @@ public Collection ensureCollection(String unitName, String name) {
 		Unit unit = ensureUnit(unitName);
     	collection.setUnit(unit);
     	collection.setName(name);
+		collection.setActive(false);
```

Comment:
by default a newly created collection is set to active. ensureCollection is likely used elsewhere with this assumption. if you set it to false, it could break other tests. I suggest you either create another method (say, ensureInactiveCollection) or just set active to false within the test for inactive collection.

In general please be aware that changes in code could break existing tests, so we better run all tests locally to make sure they pass before PR merge. If some tests can't be run on your local env (due to remote file system, remote galaxy etc), double check bamboo log after merge to fix any failing tests.

Reply:
Yea there is no test cases using collection status, I have verified it. anyways i will create my own method